index.js

```diff
@@ -660,17 +660,18 @@ function flameGraph (opts) {
     // - Size the image up N× using attributes
     // - Squash it down N× using CSS
     // - Scale the context so 1px in all subsequent draw operations means Npx
-  
+
+    devicePixelRatio = window.devicePixelRatio
+
+    // let's resize the canvas
+    canvas
+      .style('width', w + 'px')
+      .style('height', h + 'px')
+      .attr('width', w * devicePixelRatio)
+      .attr('height', h * devicePixelRatio)
+
     // avoiding unnecessary ops if the pixelRatio didn't change since last time
     if (devicePixelRatio !== window.devicePixelRatio) {
```

Comment:
this condition will now always fail because `devicePixelRatio` was reassigned 10 lines earlier (on line 664)

Reply:
OMG 🤭! 
Thanks for spotting that! I am actually removing that `if` as it seems just to lead to incorrect renderings anyway